businessRules/donations.js

```diff
@@ -1,15 +1,15 @@
 const { v4 } = require('uuid');
 const { createDonation } = require('../models/donations');
-const { getAllIdeas } = require('../models/ideas');
-const { getIdeas } = require('../utils/storage');
+const { findIdeaWithSameId } = require('../validations/ideas');
+const {
+    validateSum,
+    findIdeaWithTotalDonationSumWithSameId,
+} = require('../validations/donations.js');
 
-function createDonation(firstName, sum, userId, ideaId) {
-    let ideas = getIdeas();
-    const idea = ideas.find((idea) => ideaId === idea.id);
+function createDonations(firstName, sum, userId, ideaId) {
+    findIdeaWithSameId(ideaId);
```

Comment:
`find` is in the `get` category, but here the `findIdeaWithSameId` function call doesn't return anything (even if it finds it), so the name is rather confusing - can you try to improve it?

Reply:
Renamed.